requirements-integration.in

```diff
@@ -7,3 +7,4 @@ pytest-operator
 pytest-timeout
 pyyaml
 requests
+charmed-kubeflow-chisme
```

Comment:
```suggestion
# Pin to >=0.4.0 because the reusable test infrastructure is on that version and above
# This prevents pip-compile from trying to pin an earlier version
charmed-kubeflow-chisme>=0.4.0
```